File: alpro-lioneljevon-2-10-2025/segitiga.py
print("Program Menghitung Segitiga \n")

while True:
    alas = float(input("Masukkan alas: "))
    tinggi = float(input("Masukkan Tinggi: "))

    luas_segitiga = 0.5 * alas * tinggi

    print(f"Luas Segitiga adalah: {luas_segitiga} \n")
    lanjut = str(input("Lanjut (Ketik 0 untuk break): "))

    if lanjut == "0":
        break

File: alpro-lioneljevon-2-10-2025/Versi C++/segitiga.cpp
#include <iostream>
using namespace std;

int main() {
    cout << "++ Menghitung Luas Segitiga ++" << endl;

    double alas, tinggi;
    
    cout << "Masukkan Alas : "; cin>> alas;
    cout << "Masukkan Tinggi Segitiga : "; cin>> tinggi;
    
    double luas_segitiga = 0.5 * alas * tinggi;
	
	cout << "Luas Segitiga Adalah: " << luas_segitiga; 

    return 0;
}
